styleguide/screens/Patterns/Tooltip/Tooltip.js

```diff
@@ -0,0 +1,156 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import cx from 'classnames';
+import dedent from 'dedent';
+
+import Specimen from '../../../components/Specimen/Specimen';
+import { D, H, T, C } from '../../../components/Scaffold/Scaffold';
+import Tooltip from '../../../../components/Tooltip/Tooltip';
+import { HORIZONTAL_ATTACHMENTS, VERTICAL_ATTACHMENTS } from '../../../../components/Tether/Tether';
+
+import m from '../../../../globals/modifiers.css';
+import css from './Tooltip.css';
+import scaffoldCss from '../../../components/Scaffold/Scaffold.css';
+
+// eslint-disable-next-line react/prop-types
+const DefaultTarget = ({ onMouseEnter, onMouseLeave }) => (
+  <button
+    onMouseEnter={ () => { onMouseEnter('showDefaultTooltip'); } }
+    onMouseLeave={ () => { onMouseLeave('showDefaultTooltip'); } }
+  >
+    Hover on me
+  </button>
+);
+
+// eslint-disable-next-line react/prop-types
+const LightTarget = ({ onMouseEnter, onMouseLeave }) => (
```

Comment:
The target should probably be consistent with other pages and be one of the `Placeholder` components found in `Scaffold.js`, [as per the modal documentation](https://bloom.appearhere.co.uk/patterns/modals).

I'd prefer for the tooltip to show on click too, as that's the best event to use for cross device support. I'd be tempted to remove any mention of `hover` from the documentation too, but that's open for debate

Reply:
@farazcsk what are your thoughts on the following?

> I'd be tempted to remove any mention of hover from the documentation too, but that's open for debate

Reply:
@rdjpalmer So I actually already made the changes stated above as I totally agree with point made in the documentation about not using tooltips on mobile, but In hindsight I think that we should remove the mention of click 😆 .

I'm probably wrong, but I don't think tooltips were ever designed to be shown on click and mentioning that they basically won't be seen on mobile devices is probably enough.